plugins/PrivacyManager/templates/askingForConsent.twig

```diff
@@ -4,48 +4,49 @@
 
 {% block content %}
     <div piwik-content-intro>
-        <h2>Asking for consent</h2>
-        <p>Consent means offering individuals choice and control about how their personal data is processed.</p>
+        <h2>{{ 'PrivacyManager_AskingForConsent'|translate|e('html_attr') }}</h2>
+        <p>{{ 'PrivacyManager_ConsentMeans'|translate|e('html_attr') }}</p>
     </div>
 
-    <div piwik-content-block content-title="When do I need to ask for user consent?" class="privacyAskingForConsent">
+    <div piwik-content-block content-title="{{ 'PrivacyManager_WhenINeedConsent'|translate|e('html_attr') }}" class="privacyAskingForConsent">
         <p>
-            In the context of the GDPR privacy regulations, when you are processing personal data, in some cases you will need to ask for your users' consent. To identify whether you need to ask for consent, you need to determine whether your lawful basis for processing personal data is "Consent" or "Legitimate interest", or whether you can avoid collecting personal data altogether. We recommend to learn more about the lawful basis under the GDPR for Matomo.
+            {{ 'PrivacyManager_WhenINeedConsentNote1'|translate|e('html_attr') }}
             <br /><br />
-            Note that when you require to have your users' consents and if you cannot prove that you got the "Consent" of your users in the past, then we recommend to <a href="{{ linkTo({module: 'PrivacyManager', action: 'privacySettings'}) }}">anonymise your previously tracked users' raw data</a>.
+            {{ 'PrivacyManager_WhenINeedConsentNote2'|translate|e('html_attr') }} <a href="{{ linkTo({module: 'PrivacyManager', action: 'privacySettings'}) }}">anonymise your previously tracked users' raw data</a>.
         </p>
     </div>
 
-    <div piwik-content-block content-title="How do I ask users for consent?" class="privacyAskingForConsent">
+    <div piwik-content-block content-title="{{ 'PrivacyManager_HowToAskConsent'|translate|e('html_attr') }}" class="privacyAskingForConsent">
         <p>
-            Follow the steps below to ask your user for their consents before their data is processed within Matomo.
+            {{ 'PrivacyManager_HowToAskConsentNote1'|translate|e('html_attr') }}
```

Comment:
@maxpereira FYI you only need `|e('html_attr')` when it is part of an HTML attribute like `content-title="..."` above. In here for example you should remove `|e('html_attr')`